### PR TITLE
Wg/faster ci

### PR DESCRIPTION
**Pull request by willjgould**
### Prereview checklist

- [ ] The [CHANGELOG.md](../blob/master/CHANGELOG.md) has been updated under the `# Unreleased` header using the appropriate sub-headings with links to the appropriate issues/PRs.
- [x] All tests pass in CI
- [x] PR was self-reviewed

### CODE REVIEW

File: offchain/test/Test/CommitteeCandidateValidator.purs [hunk 1]
```diff
@@ -188,21 +188,12 @@ testScenarioFailure2 =
...
-            dummySidechainParams
+          withUnliftApp (Wallet.withKeyWallet alice) $ do
+            void $ runRegister dummySidechainParams
+
+            -- alice tries to register again with the same set of keys. not allowed & should fail
+            (void $ runRegisterWithFixedKeys dummySidechainParams) # withUnliftApp
+              fails
           withUnliftApp (Wallet.withKeyWallet bob) $ runDeregister
             dummySidechainParams
           # withUnliftApp fails
```
Comment (jankun4) [inline offchain/test/Test/CommitteeCandidateValidator.purs:199]: Why do we deregister, if previous command should fail?

Comment (willjgould) [reply offchain/test/Test/CommitteeCandidateValidator.purs:199]: Also reminder to myself update comments

Comment (willjgould) [reply offchain/test/Test/CommitteeCandidateValidator.purs:199]: To actually answer the review comment, I think the failure doesn't cause the test to quit. We just check that it did fail then continue. But this needs verifying.

Comment (willjgould) [reply offchain/test/Test/CommitteeCandidateValidator.purs:199]: I confirmed this ^

**Review by jankun4 — APPROVED** [review]

**Pull request merged**